docs/src/index.md

```diff
@@ -0,0 +1,46 @@
+---
+name: Getting Started
+---
```

Comment:
```suggestion
route: /
---
```

Reply:
We can define the page route too, have a look if this one works (if it doesn't work after deploying we might need to take the base path into account - `/opensource/cdflow2/`)